src/www/webpack_cordova.js

```diff
@@ -16,7 +16,7 @@ const path = require('path');
 const CopyPlugin = require('copy-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
-const {makeConfig} = require('./base.webpack.js');
+const {makeConfig} = require('./webpack_base.js');
```

Comment:
What would you say is the difference between `main` and `base` as far as webpack is concerned?

Reply:
There's no "main". It's "electron_main", meaning that's building the component to run on the Electron main process (Node.js). The target is called `electron-main` on webpack.

Reply:
Ah, I wish there was a way to distinguish this. Like "electron_main-webpack" or "electron_main__webpack"